tests/defaults_list/test_defaults_list.py

```diff
@@ -670,6 +670,79 @@ def test_include_nested_group_global(
     )
 
 
+@mark.parametrize(
+    "config_name, overrides, expected",
+    [
+        param(
+            "group_default_at_global",
+            [],
+            [
+                ResultDefault(
+                    config_path="group1/file1",
+                    package="",
+                    parent="group_default_at_global",
+                ),
+                ResultDefault(
+                    config_path="group_default_at_global",
+                    package="",
+                    is_self=True,
+                ),
+            ],
+            id="group_default_at_global",
+        ),
+        param(
+            "group_default_at_global",
+            ["+experiment=override_with_global_default2"],
+            [
+                ResultDefault(
+                    config_path="group1/file2",
+                    package="",
+                    parent="group_default_at_global",
+                ),
+                ResultDefault(
+                    config_path="group_default_at_global",
+                    package="",
+                    is_self=True,
+                ),
+                ResultDefault(
+                    config_path="experiment/override_with_global_default2",
+                    package="experiment",
+                    parent="group_default_at_global",
+                ),
+            ],
+            id="include_absolute_config:override_with_global_default2",
+        ),
+        param(
+            "two_group_defaults_different_pkgs_global",
+            [],
+            [
+                ResultDefault(
+                    config_path="group1/file1",
+                    parent="two_group_defaults_different_pkgs_global",
+                    package="group1",
+                ),
+                ResultDefault(
+                    config_path="group1/file2",
+                    parent="two_group_defaults_different_pkgs_global",
+                    package="",
+                ),
+                ResultDefault(
+                    config_path="two_group_defaults_different_pkgs_global",
+                    package="",
+                    is_self=True,
+                ),
+            ],
+        ),
```

Comment:
missing id for this test.

Reply:
Added in https://github.com/facebookresearch/hydra/pull/1791/commits/806eb0b95082eb1e6398c2d655e189015af94361.